src/Components/IncidentList/IncidentList.scss

```diff
@@ -14,4 +14,118 @@
     &:hover {
         cursor: pointer;
     }
-}
\ No newline at end of file
+}
+
+.all-incidents {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+
+    .incidents {
+        display: flex;
+        width: 33em;
```

Comment:
Any reason for using `em` instead of `rem`?

Reply:
@Mbarak-Mbigo I've used `em` all through?
Any reason I should switch to `rem`?

Reply:
https://webdesign.tutsplus.com/tutorials/comprehensive-guide-when-to-use-em-vs-rem--cms-23984 👈